### PR TITLE
Very minor changes to some Python files

### PR DESCRIPTION
**Pull request by RyDroid**

### CODE REVIEW

File: src/Classifier/JSClassifier.py [hunk 1]
```diff
@@ -16,7 +16,6 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston,
 # MA  02111-1307  USA
 
-import os
```
Comment (RyDroid) [inline src/Classifier/JSClassifier.py]: Useless

File: src/Classifier/SampleClassifier.py [hunk 2]
```diff
@@ -16,15 +16,14 @@
...
-# Original code written by Thorsten Sick <thorsten.sick@avira.com> 
+# Original code written by Thorsten Sick <thorsten.sick@avira.com>
 # from Avira (developed for the iTES Project http://ites-project.org)
 #
 # Modified by Angelo Dell'Aera:
 #   - Designed the more generic Classifier module and embedded this
 #     module into such module
 #   - Converted to YARA rules
 
-import os
```
Comment (RyDroid) [inline src/Classifier/SampleClassifier.py]: Useless

File: tools/distributed/thugd.py [hunk 3]
```diff
@@ -151,7 +154,7 @@ def process(self, job):
 
         for line in self.runProcess(command):
             if line.startswith("["):
-                print line,
+                print(line)
```
Comment (RyDroid) [inline tools/distributed/thugd.py]: Python 3 compatibility

**Comment by buffer** [comment]
Please clean up the patch before committing. You just deleted 3 lines and ended up modifying 8 files.

**Comment by RyDroid** [comment]
I deleted 2 lines with useless import and 1 empty at the end of a file. What is wrong with that? Do you prefer a commit per directory?

**Pull request closed without merging**